New submission: GECKO (SARIMA) model for Hospital Admission Forecasting

mpanaggio opened the pull request:
## Description

Please include a brief description of your change.  

---
> _If this section does not apply to you, please delete it in your PR_  

If this is a **new team submission**, please include the following details :-  
- Team name: JHUAPL
- Model Name: GECKO
- Institution:  Johns Hopkins University Applied Physics Laboratory
---

## Checklist

- [ x] Specify a proper PR title with your team name.
- [ x] All validation checks ran successfully on your branch. Instructions to run the tests locally is present [here](https://github.com/reichlab/covid19-forecast-hub/wiki/Running-Checks-Locally).

nickreich commented:
Please remove the conflicting file from the PR. Thanks!

nickreich commented:
Can you please remove from the PR the two changed files in the code/validation folder? If you think the change to the .py script is an improvement, please submit that as a separate PR.

nickreich commented:
Also, since this is a different model to the other models already being submitted by JHUAPL, should we list it as "secondary" rather than "other" in the designation in the metadata?

mpanaggio commented:
Yes, I can do that.  The change to the script was needed to avoid an error when running on a windows machine. I can resubmit it later on.

mpanaggio commented:
I added this week's forecast, changed the metadata to use secondary rather than other, and reverted to the original version of the conflicting files as requested.  Thanks for your patience with this MR! Hopefully now that I am familiar with the process, the next one will go more smoothly.

nickreich commented on code/validation/locally_validated_files.csv:
```diff
@@ -2305,3 +2305,17 @@ file_path,validation_date
 ./data-processed/Caltech-CS156/2021-01-17-Caltech-CS156.csv,2021-01-19 06:44:44.447677
 ./data-processed/CU-select/2021-01-17-CU-select.csv,2021-01-19 06:44:59.621536
 ./data-processed/UCF-AEM/2021-01-17-UCF-AEM.csv,2021-01-19 06:44:59.759076
+./data-processed/JHU_IDD-CovidSP/2021-01-17-JHU_IDD-CovidSP.csv,2021-01-19 18:18:33.969454
```
please remove the added lines from these files. thanks

nickreich commented:
Ok - there still are some added lines in the `code/validation/locally_validated_files.csv`. please remove these lines from the file in the PR.

Also, in general we don't allow teams to submit files from weeks past. however, when a model is making an initial submission, we have allowed previous weeks to be submitted if the forecasts were legitimately generated in real-time in past weeks. Please either (a) remove the previous weeks' files from the commit or (b) leave them in and attest to the fact that they were generated in real time.

Thanks!

mpanaggio reviewed (commented):
Strange, that those lines did not match up since I replaced the file with the same one in the original repo, However, it should match up now.
